old/lib/LedgerSMB/DBObject/Reconciliation.pm

```diff
@@ -377,10 +378,10 @@ sub get {
     my %report_days = map { $_->{id} => $_->{days} } @{$db_report_days};
     ($ref) = $self->call_dbmethod(funcname=>'account_get',
                                 args => { id => $self->{chart_id} });
-    my $neg = 1;
+    my $pos = 1;
     if (defined $self->{account_info}->{category}   # Report may be empty
     and $self->{account_info}->{category} =~ /(A|E)/){
-        $neg = -1;
+        $pos = 0;
```

Comment:
As suggested previously we really should look at factoring this sort of behaviour put to a single function if possible